Obstacle script

BerkantA21 opened the pull request:
Obstacle script werkt alleen voor saw. Later zal ik meer Obstacle's erbij toevoegen.

BerkantA21 committed:
obstacle script aangemaakt

Moet alleen nog de damage van de obstacle bewerken

BerkantA21 committed:
Commit Sideways_script updated and done.

Saw werkt compleet. Player gaat dood wanneer hij de Saw aanraakt.

BerkantA21 committed:
Update level 1 unity

Heb de level 1 geupdated zodat de saw damage kan doen tegen over de player

Thomas1509 reviewed (approved):
Alles werkt, mooie code

Jaswand reviewed (approved):
Code ziet er uitstekend eruit

Meessus reviewed (approved):
Mooie code!

BerkantA21 committed:
commit deleted animation

Moest even de animatie verwijderen van player omdat we even de conflict aan het fixen zijn.

BerkantA21 committed:
Merge branch 'Development' into Obstacle-Script
